cdap-ui/cypress/helpers.ts

```diff
@@ -115,4 +115,15 @@ function deployAndTestPipeline(filename, pipelineName, done) {
     .then(() => done());
 }
 
-export { loginIfRequired, getArtifactsPoll, deployAndTestPipeline, getSessionToken };
+function getGenericEndpoint(options, id) { return `.plugin-endpoint_${id}-right` }
+
+function getConditionNodeEndpoint(options, id) { return `.plugin-endpoint_${id}_condition_${options.condition}` }
```

Comment:
should these be `.plugin-endpoint-${id}`? dash instead of underscore?

Reply:
Actually no. I see why this is confusing. `data-cy` attribute on plugin endpoint dom that we use has that pattern `plugin-endpoint-${id}` but when here we are querying for css class. When we create endpoints for different nodes we add a css class of the form `plugin-${domid}` where the domid is of the form `endpoint_<uuid>`. I will remove the unnecessary data-cy attributes to avoid confusion